Add info on Common JSON schema blocking (#5825)
segmentio/segment-docs: src/protocols/enforce/schema-configuration.md

>
> To view all archived events, go to your **Source Schema** page, click **Filter** next to the search bar, and select **Archived**. To unarchive events that have been archived, click **Unarchive** in the event column. 

## Order of Priority in Blocking Options

> When setting up Schema Configuration, please note the following prioritization of blocking controls:

Reviewer: ```suggestion
When setting up Schema Configuration, note that Segment prioritizes blocking controls in the following order:
```

> 1. **Standard Schema Controls:** Initially, the system evaluates incoming events against these controls, along with your Tracking Plan. Events, properties, or traits not blocked or omitted in this phase then proceed to the next level of controls: 

Reviewer: ```suggestion
1. **Standard Schema Controls**: Segment first evaluates incoming events against these controls and your Tracking Plan. Events, properties, or traits not blocked or omitted in this phase then flow to the next level of controls: the Advanced Blocking Controls/Common JSON Schema. 
```

> ![image](https://github.com/segmentio/segment-docs/assets/78318468/6610f54e-6a09-4252-88b9-6be414d78065)

Reviewer: hi @ashton-huxtable! For future PRs, are you able to upload the original files into the src/protocols/images folder? We try to have copies of each image in our regular file tree!

```suggestion
![A screenshot of the Uplanned Events, Properties and Values table, which contains unplanned events, unplanned properties/traits, and JSON schema violation columns.](../images/standard-schema-controls.png)
```

Reviewer: sounds good, thank you! @ashton-huxtable

> 2. **Advanced Blocking Controls/Common JSON Schema:** These controls are assessed after the Standard Schema Controls. They act as an essential secondary layer, evaluating incoming events against the Common JSON schema included in your Tracking Plan: 

Reviewer: ```suggestion
2. **Advanced Blocking Controls/Common JSON Schema:** These controls act as a secondary layer, evaluating incoming events against the Common JSON schema included in your Tracking Plan.
````

> ![image](https://github.com/segmentio/segment-docs/assets/78318468/f12f8bd4-03f7-48bd-ab6f-1c7d85b7accf)

Reviewer: ```suggestion
![A screenshot of the Advanced Blocking Controls table, which contains two columns: call type and common JSON schema violations.](../images/advanced-blocking-controls.png)
```

> info "Using only the Common JSON Schema to block events"

> > If a tracking plan is empty of events, but has Common JSON Schema rules, you do not need to use the standard schema controls. You would only need to use the Advanced Blocking Controls in your source. If you use the standard schema controls and omit properties or traits that do not exist, the Tracking Plan may not generate violations for the Common JSON Schema, as the entire Tracking Plan has nothing and everything will be considered unplanned. 

Reviewer: ```suggestion
> info "Using only the Common JSON Schema to block events"
> If your Tracking Plan only has Common JSON Schema rules, you only need to use the Advanced Blocking Controls for your source. 
>
> If you use the Standard Schema Controls and omit properties or traits that do not exist, the Tracking Plan might not generate violations for the Common JSON Schema, as the entire Tracking Plan has nothing and everything is considered to be "unplanned". 
```

## Order of Priority in Blocking Options

When setting up Schema Configuration, note that Segment prioritizes blocking controls in the following order:

1. **Standard Schema Controls**: Segment first evaluates incoming events against these controls and your Tracking Plan. Events, properties, or traits not blocked or omitted in this phase then flow to the next level of controls: the Advanced Blocking Controls/Common JSON Schema. 
![A screenshot of the Uplanned Events, Properties and Values table, which contains unplanned events, unplanned properties/traits, and JSON schema violation columns.](../images/standard-schema-controls.png)

2. **Advanced Blocking Controls/Common JSON Schema:** These controls act as a secondary layer, evaluating incoming events against the Common JSON schema included in your Tracking Plan.
![A screenshot of the Advanced Blocking Controls table, which contains two columns: call type and common JSON schema violations.](../images/advanced-blocking-controls.png)

> > info "Using only the Common JSON Schema to block events"

Reviewer: ```suggestion
```
